packages/nimble-components/src/table/components/group-row/styles.ts

```diff
@@ -17,13 +17,14 @@ import { themeBehavior } from '../../../utilities/style/theme';
 import { userSelectNone } from '../../../utilities/style/user-select';
 
 export const styles = css`
-    ${display('flex')}
+    ${display('grid')}
 
     :host {
         align-items: center;
         height: calc(${controlHeight} + 2 * ${borderWidth});
         border-top: calc(2 * ${borderWidth}) solid ${applicationBackgroundColor};
         box-sizing: border-box;
+        grid-template-columns: var(--ni-private-table-group-row-grid-columns) auto;
```

Comment:
Can we leave off the imperative controlHeight evaluation in JS and instead use the css variable here?
Doesn't seem to need to calculate anything on it, just reference it.

```suggestion
        grid-template-columns: ${controlHeight} var(--ni-private-table-group-row-grid-columns) auto;
```

Reply:
While I was able to do what you suggest, it turns out I needed the imperative evaluation of the `controlHeight` for a different CSS variable that was setting that was conditional. Open to other suggestions of course.